### DISCUSSION TITLE
Seeing execution time as 00:00:00

### DISCUSSION THREAD

**Discussion by bansal63** [getgauge/gauge#2112] [General]
Hi Team,

For some specs we are getting execution time correctly.

But for some we are seeing execution time as 00:00:00
We see execution as 00:00:00 for spec, at scenario and at every step.

> Gauge version: 1.1.1
> Plugins
> -------
> csharp (0.10.6)
> html-report (4.0.12)
> java (0.7.3)
> json-report (0.3.3)
> ruby (0.5.4)
> screenshot (0.0.1)
> xml-report (0.2.3)

**Comment by zabil** [comment]
Can you try upgrading to the latest version of Gauge and Html report which are 1.4.1, 4.1.2?
